deployment/k8s/charts/aggregates/migrationAssistantWithArgo/templates/resources/s3/snapshotConfigMap.yaml

```diff
@@ -3,14 +3,14 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: s3-snapshot-config
+  name: migrations-default-s3-config
   labels:
     workflows.argoproj.io/configmap-type: Parameter
 data:
-  AWS_REGION: {{ .Values.snapshotBucketConfiguration.region }}
-  {{- if .Values.snapshotBucketConfiguration.useLocalStack }}
-  ENDPOINT: "http://{{ .Values.snapshotBucketConfiguration.endpoint }}:4566"
+  AWS_REGION: {{ .Values.aws.region }}
```

Comment:
the required keyword below makes me ask, what happens if this region value isn't present?  What will the user see?